dotrix_primitives/src/cube.rs

```diff
@@ -22,24 +22,28 @@ impl Cube {
     }
 
     /// Add vertices to mesh
+    #[must_use]
```

Comment:
I did not knew this macro exists o_O

Reply:
Clippy taught me it. Clippy insists on it in this case because we are returning self. I don't remember clippy doing this before but I've just updated my nightly so it might be a new clippy lint.